docs/source/overview/quicktour.md

```diff
@@ -122,8 +102,7 @@ Finally, you can select Olive features such as which performance metrics you wan
 
 ```json
     "engine": {
-        "log_severity_level": 0,
-        "evaluator": "my_evaluator"
```

Comment:
better not remove these doc stuff?

Reply:
just remove those in example json.

Reply:
Maybe we can remove as evaluator is not required be default. We target to show quickest and simplest docs. 
@devang-ml any ideas?

Reply:
Quick tour is meant for the smallest example possible, so no need to keep non-default evaluator here.